app/models/Campaign.coffee

```diff
@@ -15,6 +15,11 @@ module.exports = class Campaign extends CocoModel
   initialize: (options = {}) ->
     @forceCourseNumbering = options.forceCourseNumbering
     super(arguments...)
+    
+  @getLevels: (campaign) ->
```

Comment:
Pretty confusing to have static and dynamic `getLevels()` without them being more related.

E.g. do they return the same types?  Unclear without looking at usage elsewhere.

Reply:
Good point, refactored to have the dynamic methods use the static ones. Basically one uses Backbone models, the other uses raw JS data.